sdk/python/kubeflow/training/api_client.py

```diff
@@ -12,22 +12,21 @@
 
 import atexit
 import datetime
-from dateutil.parser import parse
 import json
 import mimetypes
-from multiprocessing.pool import ThreadPool
 import os
 import re
 import tempfile
+from multiprocessing.pool import ThreadPool
 
+import kubeflow.training.models
 # python 2 and python 3 compatibility library
 import six
-from six.moves.urllib.parse import quote
-
-from kubeflow.training.configuration import Configuration
-import kubeflow.training.models
+from dateutil.parser import parse
 from kubeflow.training import rest
-from kubeflow.training.exceptions import ApiValueError, ApiException
+from kubeflow.training.configuration import Configuration
+from kubeflow.training.exceptions import ApiException, ApiValueError
+from six.moves.urllib.parse import quote
```

Comment:
Please can you remove changes from api_client.py file since this file is generated automatically by CODE_GEN: https://github.com/kubeflow/training-operator/blob/master/hack/python-sdk/gen-sdk.sh#L48.